test/e2e/framework/runtimeclass_util.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
preferably the new file should be under `runtimeclass/util.go`

Reply:
Thanks @neolit123.  This file already exists. I didn't want to put it into node/runtimeclass_util, since it is used in scheduler as well.

Reply:
/cc @oomichi WDYT?

Reply:
/cc @oomichi

Reply:
+1 for putting the parts of this that are used in multiple test pkgs in `node/runtimeclass.go`

Reply:
Just to clarify, we are talking about e2e/framework/node/runtimeclass.go, not e2e/node/runtimeclass.go.  

sgtm